Example/Example Swift/ComponentAPICoordinator.swift

```diff
@@ -317,6 +335,16 @@ extension ComponentAPICoordinator {
     fileprivate func delete(document: GiniVisionDocument) {
         documentService?.delete(document)
     }
+    
+    private func showErrorInAnalysisScreen(with message: String,
+                                           for documentRequest: DocumentRequest,
+                                           action: @escaping () -> Void) {
+        if self.analysisScreen == nil {
+            self.analysisScreen = AnalysisViewController(document: documentRequest.document)
```

Comment:
If you have to create an analysis screen, don't you also have to put in on screen?

Reply:
I'm not super convince about this thing with the `AnalysisViewController`, but the other thing would be to keep a reference to the analysis error and passing it to the view controller which I don't like either although maybe it is better. What do you think?